app/views/users/index.html.erb

```diff
@@ -1,2 +1,16 @@
-<h1>List Users </h1>
-<p>List of all users will be displayed here</p>
+<h1>List all the Users </h1>
+<main id="home" class="page">
+  <ul class="users-list">
+<% @users.each do |user| %>
+<li class="user-container">
+ <div class="image-div">
+ <%= image_tag('user.png', :width => "100px", :heigth=>"100px") %>
+ </div>
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be very nice if you show a picture of a registered person from the database photo column 👍 to give your user a more real-life app experience 👍